src/Unleash/Scheduling/FetchFeatureTogglesTask.cs

```diff
@@ -78,7 +78,7 @@ public async Task ExecuteAsync(CancellationToken cancellationToken)
             } 
             catch (IOException ex)
             {
-                Logger.ErrorException($"UNLEASH: Unhandled exception when writing to toggle file '{toggleFile}'.", ex);
+                Logger.WarnException($"UNLEASH: Unhandled exception when writing to toggle file '{toggleFile}'.", ex);
```

Comment:
Seems pretty handled to me, this might be a nice change while we're in the area

```suggestion
                Logger.WarnException($"UNLEASH: Exception when writing to toggle file '{toggleFile}'.", ex);
```

Reply:
Agreed!